src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -592,6 +592,14 @@ public boolean requiresIntervals() {
         return false;
     }
 
+    /**
+     * Does this tool want to disable the progress meter? If so, override here to return true
+     * 
+     * @return true if this tools wants to disable progress meter output, otherwise false
+     */
+    public boolean disableProgressMeter() {
```

Comment:
There are some issues with integrating this new method into the various traversals, but I will address those myself in a separate PR after this is merged.